internal/blueprint/ca_customizations.go

```diff
@@ -0,0 +1,5 @@
+package blueprint
+
+type CACustomization struct {
```

Comment:
This maybe out of scope for this PR but we have exactly this struct define in "images" and it is public, couldn't we just import it instead of copying it (the same question for most of the others but I guess this is why it's out of scope).

Reply:
I have no idea why is it like that, I am just following the pattern here. I am running a research parallel to this to find a common blueprint that should solve this problem for once. I would say let’s not change this in this PR?